src/stdlibs/LinearAlgebra.jl

```diff
@@ -397,4 +397,24 @@ function LinearAlgebra._kron!(C::AnyTracedRMatrix, A::AnyTracedRMatrix, B::AnyTr
     return C
 end
 
+function LinearAlgebra.axpy!(α::Number, x::TracedRArray{T}, y::TracedRArray{T}) where {T}
+    if length(x) != length(y)
+        throw(DimensionMismatch(lazy"x has length $(length(x)), but y has length $(length(y))"))
+    end
+    ax = Ops.multiply(x, TracedUtils.broadcast_to_size(T(α), size(x)))
+    
+    set_mlir_data!(y, get_mlir_data(Ops.add(y, ax)))
+end
```

Comment:
the base method also returns `y`

```suggestion
    return y
end
```